lib/SparkPost/MessageEvent.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace SparkPost;
+
+/**
+ * SDK class for querying the Message Events API
+ *
+ * @see https://developers.sparkpost.com/api/#/reference/message-events
+ */
+class MessageEvent extends APIResource
```

Comment:
We should probably call it MessageEvents since interactions with this resource will always be a collection.